typescript/cli/src/logger.ts

```diff
@@ -1,9 +1,24 @@
 import chalk, { ChalkInstance } from 'chalk';
 import { pino } from 'pino';
 
-import { isLogPretty, rootLogger } from '@hyperlane-xyz/utils';
+import {
+  LogFormat,
+  LogLevel,
+  configureRootLogger,
+  getLogFormat,
+  rootLogger,
+  safelyAccessEnvVar,
+} from '@hyperlane-xyz/utils';
+
+let logger = rootLogger.child({ module: 'cli' });
+
+export function configureLogger(logFormat: LogFormat, logLevel: LogLevel) {
+  logFormat =
+    logFormat || safelyAccessEnvVar('LOG_FORMAT', true) || LogFormat.Pretty;
+  logLevel = logLevel || safelyAccessEnvVar('LOG_LEVEL', true) || LogLevel.Info;
+  logger = configureRootLogger(logFormat, logLevel).child({ module: 'cli' });
```

Comment:
It was a precaution in case something logs before the yargs middleware runs but it's probably sufficient to just use rootLogger itself for that case. I'll change it